spec/world_spec.rb

```diff
@@ -5,58 +5,100 @@
 
   let (:world) {World.new}
 
-  it "returns no live cells if there is only 1 cell" do
-    cell = Cell.new([1,1])
-    world = World.new([cell])
+  describe "applies the first 3 rules concerning live cells" do
```

Comment:
I've mentioned in describe that the first describe block  is 'concerning alive cells' and the last 'concerning dead cells': is this not good enough? Yes, it looked weird to me too having a describe block with less tests. I kept them regardless as I was curious on your comments about that :)